src/hooks/useGetActiveAccount.ts

```diff
@@ -1,249 +1,92 @@
-import { SENSITIVE_STORAGE_KEYS, STORAGE_KEYS } from "config/constants";
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { navigationRef } from "components/App";
+import { STORAGE_KEYS } from "config/constants";
 import { logger } from "config/logger";
-import { Account } from "config/types";
-import { decryptDataWithPassword } from "helpers/encryptPassword";
+import { Account, HashKey } from "config/types";
+import { useAuthenticationStore } from "ducks/auth";
 import { t } from "i18next";
-import { useState, useCallback } from "react";
-import { clearTemporaryData, getHashKey } from "services/storage/helpers";
-import {
-  dataStorage,
-  secureDataStorage,
-} from "services/storage/storageFactory";
+import { useCallback, useEffect } from "react";
+import { dataStorage } from "services/storage/storageFactory";
 
 /**
- * Interface for temporary store data structure
+ * Checks if a hash key is expired
  */
-interface TemporaryStore {
-  expiration: number;
-  privateKeys: Record<string, string>;
-  mnemonicPhrase: string;
-}
+export const isHashKeyExpired = (hashKey: HashKey): boolean =>
+  Date.now() > hashKey.expiresAt;
 
 /**
- * Retrieves data from the temporary store
+ * Gets the public key of the active account. Used on lock screen.
  */
-const getTemporaryStore = async (): Promise<TemporaryStore | null> => {
+export const getActiveAccountPublicKey = async (): Promise<string | null> => {
```

Comment:
is this `getActiveAccountPublicKey` a duplicate from the `getActiveAccountPublicKey` in ducks/auth? should we leave it only in ducks/auth?